### PR TITLE
Add new volunteer Graham Zemel

### PR DESCRIPTION
**Pull request by grahamzemel**

### CODE REVIEW

File: data/users/GrahamZemel.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
+  "name": "Graham Zemel",
+  "bio": "Full-Stack Developer - grahamzemel.com",
+  "image": "https://grahamzemel.com/logo.webp",
+  "categories": [
+    {
+      "slug": "programming",
+      "topics": ["javascript", "svelte", "tailwind", "python"]
+    }
```
Comment (codewithvoid) [inline data/users/GrahamZemel.json:9]: add , here

File: data/users/GrahamZemel.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
+{
```
Comment (codewithvoid) [inline data/users/GrahamZemel.json:1]: keep filename in snake case as in github username

File: data/users/GrahamZemel.json [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+  "name": "Graham Zemel",
+  "bio": "Full-Stack Developer - grahamzemel.com",
```
Comment (codewithvoid) [inline data/users/GrahamZemel.json:3]: No links in bio. Will give separate section for that later.

File: data/users/GrahamZemel.json [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+  "image": "https://grahamzemel.com/logo.webp",
```
Comment (codewithvoid) [inline data/users/GrahamZemel.json:4]: Do you have any image with pfp?

File: data/users/GrahamZemel.json [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+  "categories": [
+    {
+      "slug": "programming",
+      "topics": ["javascript", "svelte", "tailwind", "python"]
```
Comment (codewithvoid) [inline data/users/GrahamZemel.json:8]: nit: limit these to 2-3

**Comment by RakeshSangem** [comment]
@codewithvoid take a look into new changes

**Comment by codewithvoid** [comment]
@grahamzemel change looks good..just fix the casing in filename `GrahamZemel` > `grahamzemel`

**Comment by codewithvoid** [comment]
Thanks @grahamzemel

**Review by codewithvoid — APPROVED** [review]

**Pull request merged**